src/pages/PullRequestPage/PullCoverage/routes/FilesChangedTab/FilesChanged/FilesChangedTable/FilesChangedTable.spec.tsx

```diff
@@ -0,0 +1,611 @@
+          },
+        },
+        updatestamp: '2024-01-12T12:56:18.912860',
+        behindBy: 82367894,
+        behindByCommit: '1798hvs8ofhn',
+        comparedTo: {
+          commitid: '2d6c42fe217c61b007b2c17544a9d85840381857',
+          uploads: {
+            totalCount: 1,
+          },
+        },
+      },
+    },
+  },
+})
+
+const server = setupServer()
+
+const wrapper =
+  (queryClient: QueryClient): React.FC<React.PropsWithChildren> =>
+  ({ children }) =>
+    (
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter initialEntries={['/gh/codecov/test-repo/pull/s2h5a6']}>
+          <Route path="/:provider/:owner/:repo/pull/:pull">{children}</Route>
+        </MemoryRouter>
+      </QueryClientProvider>
+    )
+
+beforeAll(() => {
+  server.listen()
+})
+
+beforeEach(() => {
+  server.resetHandlers()
+})
+
+afterEach(() => {
+  server.resetHandlers()
+})
+
+afterAll(() => {
+  server.close()
+})
+
+describe('FilesChangedTable', () => {
+  function setup({
+    overrideComparison,
+  }: { overrideComparison?: PullComparison } = {}) {
+    const user = userEvent.setup()
+    const mockVars = jest.fn()
+
+    const queryClient = new QueryClient({
+      defaultOptions: {
+        queries: {
+          retry: false,
+        },
+      },
+    })
+
+    server.use(
+      graphql.query('Pull', (req, res, ctx) => {
+        mockVars(req.variables?.filters)
+
+        return res(ctx.status(200), ctx.data(mockPull({ overrideComparison })))
+      })
+    )
+
+    return { user, mockVars, queryClient }
+  }
+
+  describe('renders header', () => {
+    it('renders name column', async () => {
+      const { queryClient } = setup()
+      render(<FilesChangedTable />, { wrapper: wrapper(queryClient) })
+
+      const nameHeader = await screen.findByText('Name')
+      expect(nameHeader).toBeInTheDocument()
+    })
+
+    it('renders missed lines column', async () => {
+      const { queryClient } = setup()
+      render(<FilesChangedTable />, { wrapper: wrapper(queryClient) })
+
+      const nameHeader = await screen.findByText('Missed lines')
+      expect(nameHeader).toBeInTheDocument()
+    })
+
+    it('renders patch % column', async () => {
+      const { queryClient } = setup()
+      render(<FilesChangedTable />, { wrapper: wrapper(queryClient) })
+
+      const nameHeader = await screen.findByText('Patch %')
+      expect(nameHeader).toBeInTheDocument()
+    })
+
+    it('renders change column', async () => {
+      const { queryClient } = setup()
+      render(<FilesChangedTable />, { wrapper: wrapper(queryClient) })
+
+      const nameHeader = await screen.findByText('Change %')
+      expect(nameHeader).toBeInTheDocument()
+    })
+
+    it('renders coverage on head column', async () => {
+      const { queryClient } = setup()
+      render(<FilesChangedTable />, { wrapper: wrapper(queryClient) })
+
+      const nameHeader = await screen.findByText('Head %')
+      expect(nameHeader).toBeInTheDocument()
```

Comment:
are these all the `nameHeader` 👀

Reply:
yah lol i feel like the it describes what column we;re expecting